src/DynamoCoreWpf/ViewModels/Watch3D/IWatch3DViewModel.cs

```diff
@@ -61,6 +61,11 @@ public interface IWatch3DViewModel
         IRay GetClickRay(MouseEventArgs args);
 
         /// <summary>
+        /// Represents the name of current IWatch3DViewModel which will be saved in preference settings
+        /// </summary>
+        string PreferenceWatchName { get; }
```

Comment:
@sharadkjaiswal I should have caught this change as well, Thanks for finding it.